### PR TITLE
[Doc][Vectorized] Add vectorized execution engine docs

### PR DESCRIPTION
**Pull request by HappenLee**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: No Need)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (Yes)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yangzhg** [comment]
It is better to redraw those two images, the source of those two images is unknown

**Comment by HappenLee** [comment]
> It is better to redraw those two images, the source of those two images is unknown

had set quoted from [Column-Oriented
Database Systems](https://web.stanford.edu/class/cs346/2015/notes/old/column.pdf

File: docs/zh-CN/administrator-guide/vectorized-execution-engine.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+2. DateTime类型不支持秒级别以下的计算或format等各种操作，向量化引擎会直接丢弃秒级别以下毫秒的计算结果。同时也不支持`microseconds_add`等，对毫秒计算的函数。
+3. 有符合类型进行编码时，`0`与`-0`在SQL执行中被认为是相等的。这可能会影响`distinct`，`group by`等计算的结果。
+4. bitmap/hll 类型在向量化执行引擎中：输入均为NULL，则输出的结果为NULL而不是0。
+
+#### b类
+1. 不支持`地理位置函数` ，包含了函数中所有以`ST_`开头的函数。具体请参考官方文档SQL函数的部分。
+2. 不支持原有行存执行引擎的`UDF`与`UDAF`。
+3. 不支持将between语句改写为复合判断语句，会导致以下报错：`BetweenPredicate needs to be rewritten into a CompoundPredicate`。
+4. 不支持`TupleIsNull`函数，可能会导致部分外连接并带有非Nullable函数计算的表达式无法得到所需的NULL值。
+5. `string/text`类型最大长度支持为1MB，而不是默认的2GB。
```
Comment (morningman) [inline docs/zh-CN/administrator-guide/vectorized-execution-engine.md:120]:
```suggestion
5. `string/text`类型最大长度支持为1MB，而不是默认的2GB。即当开启向量化引擎后，将无法查询或导入大于1MB的字符串。但如果关闭向量化引擎，则依然可以正常查询和导入。
```

File: docs/zh-CN/administrator-guide/vectorized-execution-engine.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+6. 不支持output in file的导出方式。
```
Comment (morningman) [inline docs/zh-CN/administrator-guide/vectorized-execution-engine.md:121]:
```suggestion
6. 不支持 `select ... into outfile` 的导出方式。
```

File: docs/zh-CN/administrator-guide/vectorized-execution-engine.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+7. 不支持lateral view
+8. 不支持extrenal brocker外表
```
Comment (morningman) [inline docs/zh-CN/administrator-guide/vectorized-execution-engine.md:123]:
```suggestion
8. 不支持extrenal broker外表
```

**Comment by HappenLee** [comment]
revise done

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**